DHL/JSON/9df77954-633e-4a19-8e78-a2e73e34050f.json

```diff
@@ -0,0 +1,998 @@
+                {
+                  "Id": "181beb24-9acb-4b2b-9839-3798cbc93406",
+                  "Nm": "Business Party",
+                  "Dsc": "Business Party Entity",
+                  "isImg": false,
+                  "isExcl": false,
+                  "isMan": true,
+                  "Ent": [
+                    {
+                      "Id": "206eb8bd-37df-4696-b778-7b81f3695ab7",
+                      "Nm": "Individual Customer",
+                      "Dsc": "Individual Customer Entity",
+                      "isImg": false,
+                      "isExcl": false,
+                      "isMan": true,
+                      "Ent": [
+                        {
+                          "Id": "d4c1b6ad-3a6e-442b-825d-735bacfd1bbf",
+                          "Nm": "Consent for Returned Shipment",
+                          "Dsc": "Consent for Returned Shipment Entity",
+                          "isImg": false,
+                          "isExcl": false,
+                          "isMan": true,
+                          "Ent": [
+                            {
+                              "Id": "e51fc277-a311-4724-9dba-341b2d3e8e18",
+                              "Nm": "Consent Category",
+                              "Dsc": "Consent Category Entity",
+                              "isImg": false,
+                              "isExcl": false,
+                              "isMan": true,
+                              "Att": [
+                                {
+                                  "Id": "34300d47-a91f-48ea-aca4-156d7d8d83ce",
+                                  "Nm": "Consent Category Code",
+                                  "Dsc": "The unique identificaiton of the Consent Purpose",
+                                  "AttTy": "Constant",
+                                  "DflVal": "CSTMS"
+                                },
+                                {
+                                  "Id": "682a8af2-d276-46f7-9da0-a6000eb08e4d",
+                                  "Nm": "Consent Category Name",
+                                  "Dsc": "Consent Category Name",
+                                  "AttTy": "Constant",
+                                  "DflVal": "Customs Consents"
+                                },
+                                {
+                                  "Id": "5c523074-ab59-4ffd-8dfb-9babc95d4940",
+                                  "Nm": "Consent Category Description",
+                                  "Dsc": "Consent Category Desc",
+                                  "AttTy": "Constant",
+                                  "DflVal": "Captures Customs Consents from User"
+                                }
+                              ]
+                            },
+                            {
+                              "Id": "8430861b-38c6-4e53-92a0-5a100bdd0d05",
+                              "Nm": "Returned Shipment - Radio button",
```

Comment:
its not real radion button so update it "Consent Purpose Returned Shipment - Radio button"

Reply:
Done